### PR TITLE
Apply Styles on IOS and fix some bug

### PR DESCRIPTION
**Pull request by heartjigi**
Change Splash on IOS, apply new font,
fix some error on login.js
simulator screen shot 2016 11 17 11 18 10

### CODE REVIEW

File: src/components/Login/Login.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ class Login extends Component {
     AsyncStorage.getItem('token', (err, result) => {
       if (err) {
         this.setState({ loaded: true });
-      } else if (err) {
+      } else if (result) {
```
Comment (MechanicKim) [inline src/components/Login/Login.js:35]: Travor found same thing. sorry my mistake.
Thanks to you guys, our project are getting better!
Thanks again!

**Comment by MechanicKim** [comment]
Thank you so much! LGTM

File: src/components/Login/Login.js (hunk 1 above)
Comment (ifndefdeadmau5) [reply src/components/Login/Login.js:35]: Actually, dabin found this and notify to me :)

Comment (heartjigi) [inline src/components/Login/Login.js]: Don't mind! I  did commit and not push,
about  erasing  letterSpacing: 9, and handling AccessToken null null.
but already got LGT민규, and mingyu said handle it,. so Im going to merge.
ill handle that issue ! 
have a great day thanks.

**Comment by heartjigi** [comment]
```
letterSpacing: 9,
```

**Comment by MechanicKim** [comment]
Okay! LGTM

**Comment by MechanicKim** [comment]
I gave a 'LGTM' but, conflict! ^^;
I don't know the reason.
Sorry, please solve the conflict.

**Comment by heartjigi** [comment]
solved!

**Pull request merged**